Content/FAQEmail.md

```diff
@@ -4,44 +4,44 @@ Questions and anwers regarding the use of Sandboxie with email software. For a l
 
-A. No. The updates will be installed in the sandbox and will disappear when the sandbox is [deleted](DeleteSandbox.md). To properly update your email software, run it outside the supervision of Sandboxie, then initiate the update process. If your email software is already set as a forced program (see previous question), use the [Disable Forced Programs](FileMenu.md#disable-forced-programs) command before starting your email software.
+A. No. The updates will be installed in the sandbox and will disappear when the sandbox is [deleted](DeleteSandbox.md). To properly update your software, launch it outside the supervision of Sandboxie, then initiate the update process. If your software is already set as a forced program (see previous question), use the [Disable Forced Programs](FileMenu.md#disable-forced-programs) command before starting your email software.
 
 ### Q. Should I create a separate, dedicated sandbox just for email, or can I use the same sandbox for email and web browsing?
 
-A. This depends primarily on your habits. If you want the convenience of opening your email software by clicking an email ("mailto") link in your browser, then you have to use (and configure) the same sandbox for both web browsing and email reading. On the other hand, some people prefer to isolate the two unrelated types of activities into separate sandboxes. There is no strict answer, and both approaches work well.
+A. This depends primarily on your habits. If you want the convenience of opening your email software by clicking on the email ("mailto") link in your browser, then you have to use (and configure) the same sandbox for both web browsing and email reading. On the other hand, some people prefer to isolate the two unrelated activities into separate sandboxes. There is no strict answer, and both approaches work well.
 
-### Q. I want to run my web browser in a sandbox, but not my email software. When I click an email link (_mailto_), the web browser starts my email software in the sandbox. What should I do?
+### Q. I want to launch my web browser in a sandbox, but not my email software. When I click on the email link (_mailto_), the web browser tries to launch my email software in the sandbox. What should I do?
 
-A. You can avoid this problem by right-clicking the email link instead of left (normal) clicking it. The right-click menu will let you **copy** the email address. Then switch to your email software and **paste** the email address. If the pasted email address begins with a **mailto:** prefix, then make sure to **delete** that prefix, including the colon (:).
+A. You can avoid this issue by right-clicking on the email link instead of left (normal) clicking it. The right-click menu will let you **copy** the email address. Then switch to your email software and **paste** the email address. If the pasted email address begins with a **mailto:** prefix, then make sure to **delete** that prefix, including the colon (:).

```

Comment:
```suggestion
A. You can avoid this issue by right-clicking the email link instead of left (normal) clicking it. The right-click menu will let you **copy** the email address. Then switch to your email software and **paste** the email address. If the pasted email address begins with a **mailto:** prefix, then make sure to **delete** that prefix, including the colon (:).
```